C4.puml

```diff
@@ -3,7 +3,9 @@
 'Version
 ' ##################################
 !function C4Version()
-  !return "2.5.0"
+  ' 2 spaces and ' are used as unique marker, that the release scripts makes the correct version update
```

Comment:
It might be better to use an explicit marker using a comment like:
```
/'<VERSION>'/
```

Reply:
Good idea, we can change it as soon we finalize the python scripts. But I will not fix it in this MR
(I used the same pattern with NOT replace in README.md, but there I think we have no (simple) alternative)
````
[comment]: # ("!include followed with 2 spaces that it is not replaced during release based branch updates")
```plantuml
!include  https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
```
````